packages/react-composites/src/composites/CallWithChatComposite/SidePane.tsx

```diff
@@ -89,29 +97,50 @@ export const EmbeddedPeoplePane = (props: {
   onPeopleButtonClick: () => void;
   mobileView?: boolean;
 }): JSX.Element => {
-  const { callAdapter, chatAdapter, inviteLink } = props;
+  const { callAdapter, chatAdapter, inviteLink, mobileView } = props;
   const participantListDefaultProps = usePropsFor(ParticipantList);
 
-  const callWithChatStrings = useCallWithChatCompositeStrings();
+  const locale = useLocale();
+
+  const [menuItems, setMenuItems] = useState<_DrawerMenuItemProps[]>([]);
 
-  const participantListProps = useMemo(() => {
+  const participantListProps: ParticipantListProps = useMemo(() => {
     const onRemoveParticipant = async (participantId: string): Promise<void> =>
       removeParticipantFromCallWithChat(callAdapter, chatAdapter, participantId);
     return {
       ...participantListDefaultProps,
-      onRemoveParticipant
+      onRemoveParticipant: mobileView ? undefined : onRemoveParticipant
     };
   }, [participantListDefaultProps, callAdapter, chatAdapter]);
 
   const participantList = (
     <ParticipantListWithHeading
       participantListProps={participantListProps}
       onFetchAvatarPersonaData={props.onFetchAvatarPersonaData}
-      title={callWithChatStrings.peoplePaneSubTitle}
+      title={locale.strings.callWithChat.peoplePaneSubTitle}
+      onParticipantClick={(participant?: ParticipantListParticipant) => {
+        if (participant?.userId !== participantListProps.myUserId) {
+          setMenuItems([
+            {
+              itemKey: 'remove',
+              text: locale.component.strings.participantItem.removeButtonLabel,
+              onItemClick: () => {
+                if (participant?.userId) {
+                  participantListDefaultProps.onRemoveParticipant?.(participant?.userId);
+                }
+                setMenuItems([]);
+              },
+              iconProps: {
+                iconName: 'UserRemove'
+              }
```

Comment:
This button should be disabled when the remote participant is not an ACS user. See the logic inside `ParticiantItem`.

Reply:
Good catch. Do we plan on having a `disabled` prop for `_DrawerMenuItem` like in `IContextualMenuItem` @JamesBurnside?

Reply:
If it needs it make a task and lets do it!
(Though I actually prefer the flow of the button always being enabled and if a user tries to remove a Teams user they then get a UFD-style message that they were denied due to privilages error -- this would extend better to when interop scenarios _can_ remove participants)

Reply:
I'd say let's duplicate the functionality we have so are (i.e. disabled button)

About disabled vs error message: It's best to tell the user when they can't do something as soon as we can. So if we know apriori that they can't remove a user, disable a button. If they try and then it fails, give them an error.

Reply:
I have added the `disabled` prop in this PR. The changes are summarized in this commit: https://github.com/Azure/communication-ui-library/pull/1515/commits/1b52936cfb78c21776d5c2a01af784ef25f29706.
Here is a video of how the disabled menu item looks:
https://user-images.githubusercontent.com/79475487/155943629-9ab52766-2d7c-4d8c-be52-5be63ea1bf4a.mp4